### PR TITLE
Blog post for `umapr` submitted

### PR DESCRIPTION
**Pull request by laderast**
Hi Stefanie,

Sorry this is so late in the day. Hope this is ok.

Ted

### CODE REVIEW

**Comment by stefaniebutland** [comment]
Thank you @laderast. I'll aim to publish this 2018-07-31 as the posts from the unconf education collaborative group were ready to go first. I'll do another review of this pull request before publishing

File: content/blog/2018-07-24-umapr.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+t-SNE is actually quite a slow algorithm; one of the advantages of UMAP is that it actually runs faster than t-SNE. Because the `data.frames` that are typically run with these algorithms can run into millions of rows, efficiency is important.
+
+We decided to start working on the `umapr` package to make this technique accessible within R.  As with most rOpenSci Unconf projects, this started with an [issue entry in the rOpenSci unconf repo](https://github.com/ropensci/unconf18/issues/43):
+
+> I recently read about a new non-linear dimensionality reduction algorithm called UMAP, which is much faster than t-SNE, while producing two-dimensional visualizations that share many characteristics with t-SNE. I initially found out about it in the context of use on high-dimensional single-cell data in this paper.
```
[the remark this answers is not in the document]
Comment (laderast) [reply content/blog/2018-07-24-umapr.md]: I will try and get to this tomorrow (7/31)

**Comment by laderast** [comment]
Corrections/edits have been made

**Comment by stefaniebutland** [comment]
Thank you @laderast I'll take it from here. Look for a tweet Aug 1.

**Pull request merged**